src/DataTableAbstract.php

```diff
@@ -737,6 +737,8 @@ protected function filterRecords(): void
     public function filtering(): void
     {
         $keyword = $this->request->keyword();
+        /**Following line of code to convert & or any other special character to html string. **/
+        $keyword = (is_string($keyword) || $keyword instanceof Htmlable) ? e($keyword) : $keyword;
```

Comment:
`$keyword` is always a string thus Htmlable is not needed. Maybe we can just convert to HTML string directly from the `request->keyword()` method?

Reply:
Yes, you are right. We cam ommit the conditional part and directly convert to HTML string.

Reply:
I have commited. Please take a review.